src/routes/notifications/notifications.controller.v2.spec.ts

```diff
@@ -767,7 +767,7 @@ describe('Notifications Controller V2 (Unit)', () => {
     });
 
     describe('authentication', () => {
-      it('should return 403 if no token is present', async () => {
+      it('should return "unknown" subscriptions with no token', async () => {
```

Comment:
Is this test aligned with the description? The test expects the request to fail with a 403 status if I'm not wrong.

Reply:
True, here it expects 403. Updated in da9bf557fe0afb1b1eabea3681f878250e7c90ee.